.github/workflows/unit-tests.yaml

```diff
@@ -1,9 +1,5 @@
 name: Unit Tests
 
-concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
-  cancel-in-progress: true
```

Comment:
It makes sense to delete the `concurrency` part because there is only one job in this action. Let's be consistent and remove it from `.github/workflows/lint-python.yaml` too since it also has one job.